aliases/available/dnf.aliases.bash

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#zuck007
+cite 'about-alias'
+about-alias 'dnf aliases for fedora 22+ distros'
+
+#set-dnf-aliases
+function _set_dnf_aliases() 
```

Comment:
i had also similar thought but when i see [this](https://github.com/Bash-it/bash-it/blob/master/aliases/available/apt.aliases.bash).
then i thought this is some kind of coding style of bash-it , so i just put it.:)